pom.xml

```diff
@@ -120,6 +120,8 @@
     <flink.version>1.14.3</flink.version>
     <spark2.version>2.4.4</spark2.version>
     <spark3.version>3.2.1</spark3.version>
+    <spark3.1.version>3.1.2</spark3.1.version>
+    <spark3.2.version>3.2.1</spark3.2.version>
```

Comment:
not sure why we need these new properties. `spark3.version` is always the default and point to the latest supported spark 3. and we shall build the project with `spark3.1.x` if we want `spark3.version` point to 3.1. can you clarify